Unable to build with TBB on Ubuntu

I'm trying to build CloudCompare with enabling TBB (setting `CCCORELIB_USE_TBB=ON`). Compilation completes successfully, but I'm unable to finish linking, getting undefined reference errors. Disabling building with TBB works fine, and then I'm able to use all features of CloudCompare.

### Details
<details>
<summary>Linker error output (click to expand)</summary>

```
[ 82%] Linking CXX executable CloudCompare
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccContourExtractor.cpp.o: in function `FindNearestCandidate(unsigned int&, std::_List_iterator<CCCoreLib::PointProjectionTools::IndexedCCVector2*> const&, std::_List_iterator<CCCoreLib::PointProjectionTools::IndexedCCVector2*> const&, std::vector<CCCoreLib::PointProjectionTools::IndexedCCVector2, std::allocator<CCCoreLib::PointProjectionTools::IndexedCCVector2> > const&, std::vector<HullPointFlags, std::allocator<HullPointFlags> > const&, float, bool, double) [clone .constprop.0]':
ccContourExtractor.cpp:(.text+0x1f3): undefined reference to `tbb::task_group_context::init()'
/usr/bin/ld: ccContourExtractor.cpp:(.text+0x205): undefined reference to `tbb::internal::allocate_root_with_context_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccContourExtractor.cpp:(.text+0x245): undefined reference to `tbb::internal::get_initial_auto_partitioner_divisor()'
/usr/bin/ld: ccContourExtractor.cpp:(.text+0x274): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccContourExtractor.cpp.o: in function `tbb::interface9::internal::start_for<tbb::blocked_range<unsigned int>, tbb::internal::parallel_for_body<FindNearestCandidate(unsigned int&, std::_List_iterator<CCCoreLib::PointProjectionTools::IndexedCCVector2*> const&, std::_List_iterator<CCCoreLib::PointProjectionTools::IndexedCCVector2*> const&, std::vector<CCCoreLib::PointProjectionTools::IndexedCCVector2, std::allocator<CCCoreLib::PointProjectionTools::IndexedCCVector2> > const&, std::vector<HullPointFlags, std::allocator<HullPointFlags> > const&, float, bool, double)::{lambda(unsigned int)#1}, unsigned int>, tbb::auto_partitioner const>::execute()':
ccContourExtractor.cpp:(.text+0x366): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccContourExtractor.cpp:(.text+0x38a): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccContourExtractor.cpp:(.text+0x9e1): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: ccContourExtractor.cpp:(.text+0xabe): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccContourExtractor.cpp:(.text+0xaf1): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccContourExtractor.cpp.o: in function `FindNearestCandidate(unsigned int&, std::_List_iterator<CCCoreLib::PointProjectionTools::IndexedCCVector2*> const&, std::_List_iterator<CCCoreLib::PointProjectionTools::IndexedCCVector2*> const&, std::vector<CCCoreLib::PointProjectionTools::IndexedCCVector2, std::allocator<CCCoreLib::PointProjectionTools::IndexedCCVector2> > const&, std::vector<HullPointFlags, std::allocator<HullPointFlags> > const&, float, bool, double) [clone .constprop.0] [clone .cold]':
ccContourExtractor.cpp:(.text.unlikely+0xa): undefined reference to `tbb::internal::allocate_root_with_context_proxy::free(tbb::task&) const'
/usr/bin/ld: ccContourExtractor.cpp:(.text.unlikely+0x12): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccContourExtractor.cpp.o:(.data.rel.ro._ZTIN3tbb10interface98internal9flag_taskE[_ZTIN3tbb10interface98internal9flag_taskE]+0x10): undefined reference to `typeinfo for tbb::task'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccContourExtractor.cpp.o:(.data.rel.ro+0x10): undefined reference to `typeinfo for tbb::task'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccContourExtractor.cpp.o:(.data.rel.ro._ZTVN3tbb10interface98internal9flag_taskE[_ZTVN3tbb10interface98internal9flag_taskE]+0x28): undefined reference to `tbb::task::note_affinity(unsigned short)'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccRegistrationTools.cpp.o: in function `void tbb::interface9::internal::parallel_quick_sort<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >(__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, __gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> const&)':
ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x59): undefined reference to `tbb::task_group_context::init()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0xe1): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0xed): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x15a): undefined reference to `tbb::task_group_context::init()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x176): undefined reference to `tbb::internal::allocate_root_with_context_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x19d): undefined reference to `tbb::internal::get_initial_auto_partitioner_divisor()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x1cc): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x1f0): undefined reference to `tbb::internal::allocate_root_with_context_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x21e): undefined reference to `tbb::internal::get_initial_auto_partitioner_divisor()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x26e): undefined reference to `tbb::internal::allocate_root_with_context_proxy::free(tbb::task&) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x276): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x29e): undefined reference to `tbb::internal::allocate_root_with_context_proxy::free(tbb::task&) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_[_ZN3tbb10interface98internal19parallel_quick_sortIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEEvT_SC_RKT0_]+0x2a6): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccRegistrationTools.cpp.o: in function `void tbb::interface9::internal::dynamic_grainsize_mode<tbb::interface9::internal::adaptive_mode<tbb::interface9::internal::auto_partition_type> >::work_balance<tbb::interface9::internal::start_for<tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::interface9::internal::quick_sort_body<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::auto_partitioner const>, tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> > >(tbb::interface9::internal::start_for<tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::interface9::internal::quick_sort_body<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::auto_partitioner const>&, tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >&)':
ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISG_SI_EEKNS_16auto_partitionerEEESJ_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISG_SI_EEKNS_16auto_partitionerEEESJ_EEvRT_RT0_]+0x497): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISG_SI_EEKNS_16auto_partitionerEEESJ_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISG_SI_EEKNS_16auto_partitionerEEESJ_EEvRT_RT0_]+0x7db): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISG_SI_EEKNS_16auto_partitionerEEESJ_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISG_SI_EEKNS_16auto_partitionerEEESJ_EEvRT_RT0_]+0x80a): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccRegistrationTools.cpp.o: in function `tbb::interface9::internal::start_for<tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::interface9::internal::quick_sort_body<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::auto_partitioner const>::execute()':
ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISA_SC_EEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISA_SC_EEKNS_16auto_partitionerEE7executeEv]+0x89): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISA_SC_EEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISA_SC_EEKNS_16auto_partitionerEE7executeEv]+0xb1): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccRegistrationTools.cpp.o: in function `void tbb::interface9::internal::dynamic_grainsize_mode<tbb::interface9::internal::adaptive_mode<tbb::interface9::internal::auto_partition_type> >::work_balance<tbb::interface9::internal::start_for<tbb::blocked_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > > >, tbb::interface9::internal::quick_sort_pretest_body<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::auto_partitioner const>, tbb::blocked_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > > > >(tbb::interface9::internal::start_for<tbb::blocked_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > > >, tbb::interface9::internal::quick_sort_pretest_body<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::auto_partitioner const>&, tbb::blocked_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > > >&)':
ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x1c1): undefined reference to `tbb::task::self()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x208): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x23d): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x251): undefined reference to `tbb::task::self()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x296): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x2fd): undefined reference to `tbb::task_group_context::cancel_group_execution()'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x326): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x356): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISG_St4lessIdEEEKNS_16auto_partitionerEEESH_EEvRT_RT0_]+0x411): undefined reference to `tbb::task_group_context::cancel_group_execution()'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccRegistrationTools.cpp.o: in function `tbb::interface9::internal::start_for<tbb::blocked_range<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > > >, tbb::interface9::internal::quick_sort_pretest_body<__gnu_cxx::__normal_iterator<double*, std::vector<double, std::allocator<double> > >, std::less<double> >, tbb::auto_partitioner const>::execute()':
ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISA_St4lessIdEEEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISA_St4lessIdEEEKNS_16auto_partitionerEE7executeEv]+0x8e): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: ccRegistrationTools.cpp:(.text._ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISA_St4lessIdEEEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISA_St4lessIdEEEKNS_16auto_partitionerEE7executeEv]+0xb2): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccRegistrationTools.cpp.o:(.data.rel.ro._ZTIN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISA_St4lessIdEEEKNS_16auto_partitionerEEE[_ZTIN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEEEENS1_23quick_sort_pretest_bodyISA_St4lessIdEEEKNS_16auto_partitionerEEE]+0x10): undefined reference to `typeinfo for tbb::task'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/ccRegistrationTools.cpp.o:(.data.rel.ro._ZTIN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISA_SC_EEKNS_16auto_partitionerEEE[_ZTIN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIPdSt6vectorIdSaIdEEEESt4lessIdEEENS1_15quick_sort_bodyISA_SC_EEKNS_16auto_partitionerEEE]+0x10): undefined reference to `typeinfo for tbb::task'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o: in function `MainWindow::createComponentsClouds(ccGenericPointCloud*, std::vector<CCCoreLib::ReferenceCloud*, std::allocator<CCCoreLib::ReferenceCloud*> >&, unsigned int, bool, bool, bool)':
mainwindow.cpp:(.text+0x2793a): undefined reference to `tbb::task_group_context::init()'
/usr/bin/ld: mainwindow.cpp:(.text+0x27991): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: mainwindow.cpp:(.text+0x279cb): undefined reference to `tbb::task_group_context::init()'
/usr/bin/ld: mainwindow.cpp:(.text+0x279e7): undefined reference to `tbb::internal::allocate_root_with_context_proxy::allocate(unsigned long) const'
/usr/bin/ld: mainwindow.cpp:(.text+0x27a1c): undefined reference to `tbb::internal::get_initial_auto_partitioner_divisor()'
/usr/bin/ld: mainwindow.cpp:(.text+0x27a52): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: mainwindow.cpp:(.text+0x27a5a): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: mainwindow.cpp:(.text+0x27ac0): undefined reference to `tbb::internal::allocate_root_with_context_proxy::allocate(unsigned long) const'
/usr/bin/ld: mainwindow.cpp:(.text+0x27af9): undefined reference to `tbb::internal::get_initial_auto_partitioner_divisor()'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o: in function `MainWindow::createComponentsClouds(ccGenericPointCloud*, std::vector<CCCoreLib::ReferenceCloud*, std::allocator<CCCoreLib::ReferenceCloud*> >&, unsigned int, bool, bool, bool) [clone .cold]':
mainwindow.cpp:(.text.unlikely+0x1b9e): undefined reference to `tbb::internal::allocate_root_with_context_proxy::free(tbb::task&) const'
/usr/bin/ld: mainwindow.cpp:(.text.unlikely+0x1ba6): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: mainwindow.cpp:(.text.unlikely+0x1bb5): undefined reference to `tbb::internal::allocate_root_with_context_proxy::free(tbb::task&) const'
/usr/bin/ld: mainwindow.cpp:(.text.unlikely+0x1bbd): undefined reference to `tbb::task_group_context::~task_group_context()'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o: in function `void tbb::interface9::internal::dynamic_grainsize_mode<tbb::interface9::internal::adaptive_mode<tbb::interface9::internal::auto_partition_type> >::work_balance<tbb::interface9::internal::start_for<tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::interface9::internal::quick_sort_body<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::auto_partitioner const>, tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)> >(tbb::interface9::internal::start_for<tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::interface9::internal::quick_sort_body<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::auto_partitioner const>&, tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>&)':
mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEFbRKSC_SJ_EEENS1_15quick_sort_bodyISH_SK_EEKNS_16auto_partitionerEEESL_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEFbRKSC_SJ_EEENS1_15quick_sort_bodyISH_SK_EEKNS_16auto_partitionerEEESL_EEvRT_RT0_]+0x684): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEFbRKSC_SJ_EEENS1_15quick_sort_bodyISH_SK_EEKNS_16auto_partitionerEEESL_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEFbRKSC_SJ_EEENS1_15quick_sort_bodyISH_SK_EEKNS_16auto_partitionerEEESL_EEvRT_RT0_]+0x861): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEFbRKSC_SJ_EEENS1_15quick_sort_bodyISH_SK_EEKNS_16auto_partitionerEEESL_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEFbRKSC_SJ_EEENS1_15quick_sort_bodyISH_SK_EEKNS_16auto_partitionerEEESL_EEvRT_RT0_]+0x890): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o: in function `tbb::interface9::internal::start_for<tbb::interface9::internal::quick_sort_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::interface9::internal::quick_sort_body<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::auto_partitioner const>::execute()':
mainwindow.cpp:(.text._ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEFbRKS6_SD_EEENS1_15quick_sort_bodyISB_SE_EEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEFbRKS6_SD_EEENS1_15quick_sort_bodyISB_SE_EEKNS_16auto_partitionerEE7executeEv]+0x96): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEFbRKS6_SD_EEENS1_15quick_sort_bodyISB_SE_EEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEFbRKS6_SD_EEENS1_15quick_sort_bodyISB_SE_EEKNS_16auto_partitionerEE7executeEv]+0xbf): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o: in function `void tbb::interface9::internal::dynamic_grainsize_mode<tbb::interface9::internal::adaptive_mode<tbb::interface9::internal::auto_partition_type> >::work_balance<tbb::interface9::internal::start_for<tbb::blocked_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > > >, tbb::interface9::internal::quick_sort_pretest_body<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::auto_partitioner const>, tbb::blocked_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > > > >(tbb::interface9::internal::start_for<tbb::blocked_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > > >, tbb::interface9::internal::quick_sort_pretest_body<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::auto_partitioner const>&, tbb::blocked_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > > >&)':
mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x1c1): undefined reference to `tbb::task::self()'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x20c): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x236): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x251): undefined reference to `tbb::task::self()'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x29a): undefined reference to `tbb::task_group_context::is_group_execution_cancelled() const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x305): undefined reference to `tbb::task_group_context::cancel_group_execution()'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x326): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x355): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_[_ZN3tbb10interface98internal22dynamic_grainsize_modeINS1_13adaptive_modeINS1_19auto_partition_typeEEEE12work_balanceINS1_9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorISC_SaISC_EEEEEENS1_23quick_sort_pretest_bodyISH_FbRKSC_SL_EEEKNS_16auto_partitionerEEESI_EEvRT_RT0_]+0x409): undefined reference to `tbb::task_group_context::cancel_group_execution()'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o: in function `tbb::interface9::internal::start_for<tbb::blocked_range<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > > >, tbb::interface9::internal::quick_sort_pretest_body<__gnu_cxx::__normal_iterator<ComponentIndexAndSize*, std::vector<ComponentIndexAndSize, std::allocator<ComponentIndexAndSize> > >, bool (ComponentIndexAndSize const&, ComponentIndexAndSize const&)>, tbb::auto_partitioner const>::execute()':
mainwindow.cpp:(.text._ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEEENS1_23quick_sort_pretest_bodyISB_FbRKS6_SF_EEEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEEENS1_23quick_sort_pretest_bodyISB_FbRKS6_SF_EEEKNS_16auto_partitionerEE7executeEv]+0x8e): undefined reference to `tbb::internal::allocate_continuation_proxy::allocate(unsigned long) const'
/usr/bin/ld: mainwindow.cpp:(.text._ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEEENS1_23quick_sort_pretest_bodyISB_FbRKS6_SF_EEEKNS_16auto_partitionerEE7executeEv[_ZN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEEENS1_23quick_sort_pretest_bodyISB_FbRKS6_SF_EEEKNS_16auto_partitionerEE7executeEv]+0xb2): undefined reference to `tbb::internal::allocate_child_proxy::allocate(unsigned long) const'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o:(.data.rel.ro._ZTIN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEEENS1_23quick_sort_pretest_bodyISB_FbRKS6_SF_EEEKNS_16auto_partitionerEEE[_ZTIN3tbb10interface98internal9start_forINS_13blocked_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEEENS1_23quick_sort_pretest_bodyISB_FbRKS6_SF_EEEKNS_16auto_partitionerEEE]+0x10): undefined reference to `typeinfo for tbb::task'
/usr/bin/ld: CMakeFiles/CloudCompare.dir/mainwindow.cpp.o:(.data.rel.ro._ZTIN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEFbRKS6_SD_EEENS1_15quick_sort_bodyISB_SE_EEKNS_16auto_partitionerEEE[_ZTIN3tbb10interface98internal9start_forINS1_16quick_sort_rangeIN9__gnu_cxx17__normal_iteratorIP21ComponentIndexAndSizeSt6vectorIS6_SaIS6_EEEEFbRKS6_SD_EEENS1_15quick_sort_bodyISB_SE_EEKNS_16auto_partitionerEEE]+0x10): undefined reference to `typeinfo for tbb::task'
collect2: error: ld returned 1 exit status
make[2]: *** [qCC/CMakeFiles/CloudCompare.dir/build.make:2003: qCC/CloudCompare] Error 1
make[1]: *** [CMakeFiles/Makefile2:2589: qCC/CMakeFiles/CloudCompare.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

</details>

### Reproduction
<details>
<summary><code>cmake ../ -LA</code> (click to expand)</summary>

```
-- Found Intel TBB
-- OpenMP found
-- Found 3rd party plugin: ~/src/CloudCompare/plugins/3rdParty/MeshIO
-- Found 3rd party plugin: ~/src/CloudCompare/plugins/3rdParty/qRDBIO
-- Configuring done
-- Generating done
-- Build files have been written to: ~/src/CloudCompare/build
-- Cache values
ASSIMP_ASAN:BOOL=OFF
ASSIMP_BIN_INSTALL_DIR:STRING=bin
ASSIMP_BUILD_ARCHITECTURE:STRING=
ASSIMP_BUILD_COMPILER:STRING=
ASSIMP_BUILD_ZLIB:BOOL=OFF
ASSIMP_DOUBLE_PRECISION:BOOL=OFF
ASSIMP_INCLUDE_INSTALL_DIR:STRING=include
ASSIMP_LIBRARY_SUFFIX:STRING=
ASSIMP_LIB_INSTALL_DIR:STRING=lib
ASSIMP_OPT_BUILD_PACKAGES:BOOL=OFF
ASSIMP_PACKAGE_VERSION:STRING=0
ASSIMP_UBSAN:BOOL=OFF
ASSIMP_WERROR:BOOL=OFF
BUILD_FRAMEWORK:BOOL=OFF
BUILD_TESTING:BOOL=OFF
Boost_DEBUG:BOOL=OFF
Boost_INCLUDE_DIR:PATH=/usr/include
CCACHE_PROGRAM:FILEPATH=/usr/bin/ccache
CCCORELIB_SCALAR_DOUBLE:BOOL=ON
CCCORELIB_SHARED:BOOL=ON
CCCORELIB_USE_CGAL:BOOL=OFF
CCCORELIB_USE_QT_CONCURRENT:BOOL=ON
CCCORELIB_USE_TBB:BOOL=ON
CGAL_CTEST_DISPLAY_MEM_AND_TIME:BOOL=OFF
CGAL_DEV_MODE:BOOL=OFF
CGAL_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/CGAL
CMAKE_AR:FILEPATH=/usr/bin/ar
CMAKE_BUILD_TYPE:STRING=Release
CMAKE_COLOR_MAKEFILE:BOOL=ON
CMAKE_CXX_COMPILER:FILEPATH=/usr/bin/c++
CMAKE_CXX_COMPILER_AR:FILEPATH=/usr/bin/gcc-ar-9
CMAKE_CXX_COMPILER_RANLIB:FILEPATH=/usr/bin/gcc-ranlib-9
CMAKE_CXX_FLAGS:STRING=
CMAKE_CXX_FLAGS_DEBUG:STRING=-g
CMAKE_CXX_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG
CMAKE_CXX_FLAGS_RELEASE:STRING=-O3 -DNDEBUG
CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
CMAKE_C_COMPILER:FILEPATH=/usr/bin/cc
CMAKE_C_COMPILER_AR:FILEPATH=/usr/bin/gcc-ar-9
CMAKE_C_COMPILER_RANLIB:FILEPATH=/usr/bin/gcc-ranlib-9
CMAKE_C_FLAGS:STRING=
CMAKE_C_FLAGS_DEBUG:STRING=-g
CMAKE_C_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG
CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
CMAKE_DEBUG_POSTFIX:STRING=
CMAKE_EXE_LINKER_FLAGS:STRING=
CMAKE_EXE_LINKER_FLAGS_DEBUG:STRING=
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_EXE_LINKER_FLAGS_RELEASE:STRING=
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF
CMAKE_INSTALL_BINDIR:PATH=bin
CMAKE_INSTALL_DATADIR:PATH=
CMAKE_INSTALL_DATAROOTDIR:PATH=share
CMAKE_INSTALL_DOCDIR:PATH=
CMAKE_INSTALL_INCLUDEDIR:PATH=include
CMAKE_INSTALL_INFODIR:PATH=
CMAKE_INSTALL_LIBDIR:PATH=lib
CMAKE_INSTALL_LIBEXECDIR:PATH=libexec
CMAKE_INSTALL_LOCALEDIR:PATH=
CMAKE_INSTALL_LOCALSTATEDIR:PATH=var
CMAKE_INSTALL_MANDIR:PATH=
CMAKE_INSTALL_OLDINCLUDEDIR:PATH=/usr/include
CMAKE_INSTALL_PREFIX:PATH=/usr/local
CMAKE_INSTALL_RUNSTATEDIR:PATH=
CMAKE_INSTALL_SBINDIR:PATH=sbin
CMAKE_INSTALL_SHAREDSTATEDIR:PATH=com
CMAKE_INSTALL_SYSCONFDIR:PATH=etc
CMAKE_LINKER:FILEPATH=/usr/bin/ld
CMAKE_MAKE_PROGRAM:FILEPATH=/usr/bin/make
CMAKE_MODULE_LINKER_FLAGS:STRING=
CMAKE_MODULE_LINKER_FLAGS_DEBUG:STRING=
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_MODULE_LINKER_FLAGS_RELEASE:STRING=
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_MODULE_PATH:STRING=/usr/share/cmake/Modules
CMAKE_NM:FILEPATH=/usr/bin/nm
CMAKE_OBJCOPY:FILEPATH=/usr/bin/objcopy
CMAKE_OBJDUMP:FILEPATH=/usr/bin/objdump
CMAKE_RANLIB:FILEPATH=/usr/bin/ranlib
CMAKE_SHARED_LINKER_FLAGS:STRING=
CMAKE_SHARED_LINKER_FLAGS_DEBUG:STRING=
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_SHARED_LINKER_FLAGS_RELEASE:STRING=
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_SKIP_INSTALL_RPATH:BOOL=NO
CMAKE_SKIP_RPATH:BOOL=NO
CMAKE_STATIC_LINKER_FLAGS:STRING=
CMAKE_STATIC_LINKER_FLAGS_DEBUG:STRING=
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_STATIC_LINKER_FLAGS_RELEASE:STRING=
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_STRIP:FILEPATH=/usr/bin/strip
CMAKE_VERBOSE_MAKEFILE:BOOL=FALSE
DXF_LIB_DIR:PATH=extern/dxflib
EIGEN_ROOT_DIR:PATH=/usr/include/eigen3
FFMPEG_INCLUDE_DIR:PATH=/usr/include/x86_64-linux-gnu
FFMPEG_LIBRARY_DIR:PATH=/usr/lib/x86_64-linux-gnu
HUNTER_ENABLED:BOOL=OFF
HUNTER_STATUS_DEBUG:BOOL=OFF
HUNTER_STATUS_PRINT:BOOL=ON
HUNTER_TLS_VERIFY:BOOL=ON
IGNORE_GIT_HASH:BOOL=OFF
INJECT_DEBUG_POSTFIX:BOOL=ON
NANOFLANN_ROOT_DIR:PATH=~/src/CloudCompare/plugins/core/Standard/qHoughNormals/src/nanoflann
OPTION_BUILD_CCVIEWER:BOOL=OFF
OPTION_GL_QUAD_BUFFER_SUPPORT:BOOL=OFF
OPTION_SUPPORT_3DCONNEXION_DEVICES:BOOL=OFF
OPTION_SUPPORT_GAMEPADS:BOOL=OFF
OPTION_USE_DXF_LIB:BOOL=ON
OPTION_USE_GDAL:BOOL=OFF
OPTION_USE_OCULUS_SDK:BOOL=OFF
OPTION_USE_SHAPE_LIB:BOOL=ON
OpenMP_CXX_FLAGS:STRING=-fopenmp
OpenMP_CXX_LIB_NAMES:STRING=gomp;pthread
OpenMP_C_FLAGS:STRING=-fopenmp
OpenMP_C_LIB_NAMES:STRING=gomp;pthread
OpenMP_gomp_LIBRARY:FILEPATH=/usr/lib/gcc/x86_64-linux-gnu/9/libgomp.so
OpenMP_pthread_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libpthread.so
PKG_CONFIG_EXECUTABLE:FILEPATH=/usr/bin/pkg-config
PLUGIN-3rdParty_MESH_IO:BOOL=OFF
PLUGIN_EXAMPLE_GL:BOOL=OFF
PLUGIN_EXAMPLE_IO:BOOL=OFF
PLUGIN_EXAMPLE_STANDARD:BOOL=OFF
PLUGIN_GL_QEDL:BOOL=OFF
PLUGIN_GL_QSSAO:BOOL=OFF
PLUGIN_IO_QADDITIONAL:BOOL=OFF
PLUGIN_IO_QCORE:BOOL=ON
PLUGIN_IO_QCSV_MATRIX:BOOL=OFF
PLUGIN_IO_QE57:BOOL=ON
PLUGIN_IO_QFBX:BOOL=OFF
PLUGIN_IO_QPDAL:BOOL=OFF
PLUGIN_IO_QPHOTOSCAN:BOOL=OFF
PLUGIN_IO_QRDB:BOOL=OFF
PLUGIN_STANDARD_QANIMATION:BOOL=ON
PLUGIN_STANDARD_QBROOM:BOOL=OFF
PLUGIN_STANDARD_QCANUPO:BOOL=OFF
PLUGIN_STANDARD_QCOMPASS:BOOL=OFF
PLUGIN_STANDARD_QCORK:BOOL=OFF
PLUGIN_STANDARD_QCSF:BOOL=OFF
PLUGIN_STANDARD_QFACETS:BOOL=OFF
PLUGIN_STANDARD_QHOUGH_NORMALS:BOOL=ON
PLUGIN_STANDARD_QHPR:BOOL=ON
PLUGIN_STANDARD_QM3C2:BOOL=OFF
PLUGIN_STANDARD_QPCL:BOOL=OFF
PLUGIN_STANDARD_QPCV:BOOL=OFF
PLUGIN_STANDARD_QPOISSON_RECON:BOOL=ON
PLUGIN_STANDARD_QRANSAC_SD:BOOL=OFF
PLUGIN_STANDARD_QSRA:BOOL=OFF
POISSON_RECON_WITH_OPEN_MP:BOOL=ON
QT5_ROOT_PATH:PATH=/usr/lib/x86_64-linux-gnu/qt5
Qt5Concurrent_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Concurrent
Qt5Core_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Core
Qt5Gui_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Gui
Qt5LinguistTools_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5LinguistTools
Qt5OpenGLExtensions_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5OpenGLExtensions
Qt5OpenGL_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5OpenGL
Qt5PrintSupport_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5PrintSupport
Qt5Svg_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Svg
Qt5Widgets_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets
Qt5_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5
RT_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/librt.so
SHAPELIB_LIB_DIR:PATH=extern/shapelib
SYSTEM_IRRXML:BOOL=ON
TBB_DEBUG_LIBRARY:FILEPATH=TBB_DEBUG_LIBRARY-NOTFOUND
TBB_DIR:PATH=TBB_DIR-NOTFOUND
TBB_INCLUDE_DIR:PATH=/usr/include
TBB_INCLUDE_DIRS:PATH=/usr/include
TBB_INSTALL_DIR:PATH=/opt/intel/tbb;/usr/local/include;/usr/include
TBB_LIBRARIES:PATH=optimized;/usr/lib/x86_64-linux-gnu/libtbb.so;debug;/usr/lib/x86_64-linux-gnu/libtbb.so;optimized;/usr/lib/x86_64-linux-gnu/libtbbmalloc.so;debug;/usr/lib/x86_64-linux-gnu/libtbbmalloc.so;general;-pthread
TBB_LIBRARY_DIRS:PATH=/usr/lib/x86_64-linux-gnu
TBB_MALLOCPROXY_DEBUG_LIBRARY:FILEPATH=TBB_MALLOCPROXY_DEBUG_LIBRARY-NOTFOUND
TBB_MALLOCPROXY_RELEASE_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libtbbmalloc_proxy.so
TBB_MALLOC_DEBUG_LIBRARY:FILEPATH=TBB_MALLOC_DEBUG_LIBRARY-NOTFOUND
TBB_MALLOC_RELEASE_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libtbbmalloc.so
TBB_RELEASE_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libtbb.so
WITH_FFMPEG_SUPPORT:BOOL=ON
XercesC_INCLUDE_DIR:PATH=/usr/include
XercesC_LIBRARY_DEBUG:FILEPATH=XercesC_LIBRARY_DEBUG-NOTFOUND
XercesC_LIBRARY_RELEASE:FILEPATH=/usr/lib/x86_64-linux-gnu/libxerces-c.so
ZLIB_INCLUDE_DIR:PATH=/usr/include
ZLIB_LIBRARY_DBG:FILEPATH=/usr/lib/x86_64-linux-gnu/libz.so
ZLIB_LIBRARY_REL:FILEPATH=/usr/lib/x86_64-linux-gnu/libz.so
pkgcfg_lib_UNZIP_minizip:FILEPATH=/usr/lib/x86_64-linux-gnu/libminizip.so
```

*Home directory collapsed to `~`*
</details>

Checkout master of CloudCompare repo, then build with the above configuration (I used ccmake)

### Expected behaviour
Successful build

### Environment
- CCCoreLib: bb858fa
- CloudCompare: e87f84a
- OS: Ubuntu 19.10
- Qt: 5.12.4+dfsg-4ubuntu1.1
- TBB: apt 2019~U8-1
- GCC: gcc (Ubuntu 9.2.1-9ubuntu2) 9.2.1 20191008
- Linker: GNU ld (GNU Binutils for Ubuntu) 2.33

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -162,7 +162,7 @@ endif()
 # TBB (optional)
 # Must come before CGAL so it can use TBB properly
 if ( CCCORELIB_USE_TBB )
-	find_package( TBB COMPONENTS tbb )
+	find_package( TBB COMPONENTS tbb CONFIG )
 
 	if ( TBB_FOUND )
 		target_link_libraries( CCCoreLib

